lib/services/consumed/status-bar/status-bar.js

```diff
@@ -45,19 +44,13 @@ export class StatusBarConsumer {
     return disposable;
   }
 
-  showKernelCommands({
-    kernel,
-    markers
-  }: {
-    kernel: Kernel,
-    markers: MarkerStore
-  }) {
+  showKernelCommands(store: Store, handleKernelCommand: Function) {
     let signalListView = this.signalListView;
     if (!signalListView) {
-      signalListView = new SignalListView(kernel);
-      signalListView.onConfirmed = (kernelCommand: { command: string }) =>
-        this.handleKernelCommand(kernelCommand, { kernel, markers });
+      signalListView = new SignalListView(store, handleKernelCommand);
       this.signalListView = signalListView;
+    }else{
+      signalListView.store = store
```

Comment:
please run a linter on your code, the else should have white-space around it and line 53 should have a semi-colon

Reply:
Hi @wadethestealth, thanks for review. I am not familiar with linter for javascript. I have tried installing linter-spell-javascript, linter-flow and language-babel in Atom. But none of them seems working and giving me any hint about code style. I have made changes according your comments though. Can you recommend a plugin for this please?